src/components/views/rooms/EventTile.js

```diff
@@ -215,6 +215,7 @@ module.exports = withMatrixClient(React.createClass({
         this.setState({
             verified: verified,
         });
+        this.props.onWidgetLoad();
```

Comment:
shouldn't this be in the `setState` callback, so it happens after the content is rendered?